courses: checkbox alignment (fixes #9162)

emmanuelbaa opened the pull request:
fixes #9162 

corrected the checkbox alignment on library: 

<img width="525" height="817" alt="correctedLibrary" src="https://github.com/user-attachments/assets/a984ff5b-7f70-4997-ab36-dd5f64e08fa0" />


and now it appears on courses: 

<img width="471" height="797" alt="course" src="https://github.com/user-attachments/assets/e46126e4-4cf9-4bb5-a860-59cf0582a196" />

emmanuelbaa committed:
correct checkbox misalignment (fixes #9162)

emmanuelbaa committed:
corrected the checkbox duplication (fixes #9162)

Mutugiii reviewed (changes requested):
1. This adds the title row in the mobile view of the courses. This should not be displayed as we are not using a table format in this mode

<img width="814" height="122" alt="Image" src="https://github.com/user-attachments/assets/dae2e71c-901f-4891-a061-d7a12d35def1" />

2. This PR has many formatting changes to the code. This makes it hard to follow the PR-specific changes.

google-labs-jules[bot] committed:
I have applied the `hide` class to the `mat-header-row` in `src/app/courses/courses.component.html` and verified the change.

google-labs-jules[bot] committed:
fix(courses): correct checkbox alignment and mobile header display

This commit addresses two issues in the courses component:

1.  The checkbox alignment in the mobile view has been corrected to match the alignment in the resources component.
2.  The header row in the courses table is now correctly hidden on mobile devices.

emmanuelbaa committed:
Merge branch 'master' into 9162-resources-toggleBoxInconsistency

Mutugiii reviewed (changes requested):
This PR has quite a few unrelated changes

emmanuelbaa commented:
> This PR has quite a few unrelated changes

Which changes would you suggest I undo?

emmanuelbaa committed:
correct checkbox misalignment (fixes #9162)

google-labs-jules[bot] committed:
fix(courses): correct checkbox alignment and mobile header display

This commit fixes two issues:
1.  Removes the duplicated checkbox that appears on the library and courses pages on smaller screens.
2.  Hides the table header on the courses page in mobile view, as it is not needed in that layout.

emmanuelbaa committed:
Merge branch 'master' into 9162-resources-toggleBoxInconsistency

google-labs-jules[bot] committed:
fix(courses): correct checkbox alignment and mobile header display

This commit fixes two issues:
1.  Removes the duplicated checkbox that appears on the library and courses pages on smaller screens.
2.  Hides the table header on the courses page in mobile view, as it is not needed in that layout.

google-labs-jules[bot] committed:
fix(courses): correct checkbox alignment and mobile header display

This commit fixes two issues:
1.  Removes the duplicated checkbox that appears on the library and courses pages on smaller screens.
2.  Hides the table header on the courses page in mobile view, as it is not needed in that layout.

emmanuelbaa committed:
Delete jules-scratch/verification/verify_fixes.py

Mutugiii commented on jules-scratch/verification/courses.png:
Remove this

Mutugiii commented on npm_output.log:
Remove this

emmanuelbaa committed:
updated gitignore

emmanuelbaa committed:
update gitignore

emmanuelbaa committed:
refactor: use CSS to hide select column on mobile

google-labs-jules[bot] committed:
refactor: use CSS to hide select column on mobile

Refactored the courses and resources components to use a pure CSS approach for hiding the 'select' column on mobile devices. This change addresses the feedback from the pull request review and removes the need for TypeScript logic to dynamically change the displayed columns.

- Updated `courses.scss` and `resources.scss` to hide the `mat-column-select` and `mat-header-row` on small screens.
- Removed the `setDisplayedColumns` method from the component TypeScript files.
- Removed the `npm_output.log` file from the repository.

emmanuelbaa committed:
Merge branch '9162-resources-toggle-ox-nconsistency' into 9162-resources-toggleBoxInconsistency

emmanuelbaa committed:
Merge branch 'master' of https://github.com/open-learning-exchange/planet

emmanuelbaa committed:
Fix(courses): resolve merge conflict and fix checkbox alignment

Resolved merge conflict in courses.scss that occurred when merging from the master branch. The changes from both branches were kept to fix the checkbox alignment on small screens.

emmanuelbaa committed:
made corrections to  merge conflicts in courses.scss

emmanuelbaa committed:
Merge branch 'master' of https://github.com/open-learning-exchange/planet

emmanuelbaa committed:
Merge branch 'master' of https://github.com/open-learning-exchange/planet

emmanuelbaa committed:
Delete jules-scratch/verification/resources.png

emmanuelbaa committed:
Delete jules-scratch/verification/courses.png

emmanuelbaa committed:
Delete npm_output.log

emmanuelbaa committed:
Merge branch 'master' of https://github.com/open-learning-exchange/planet

emmanuelbaa committed:
Fix(responsive): use CSS for responsive column display

Refactor display logic to use CSS for hiding columns on mobile screens, removing the need for TypeScript-based column management. This addresses the issues in PR #9187.

emmanuelbaa committed:
Fix(compiler): remove calls to deleted method

Removed calls to 'setDisplayedColumns' in 'courses.component.ts' and 'resources.component.ts' to fix compilation errors.

emmanuelbaa committed:
remove the gemini cli extension

emmanuelbaa committed:
Reformat .gitignore for consistency

emmanuelbaa committed:
removed whitespace

emmanuelbaa committed:
Merge branch '9162-resources-toggleBoxInconsistency' of https://github.com/open-learning-exchange/planet into 9162-resources-toggleBoxInconsistency

emmanuelbaa commented:
Please kindly confirm my changes.

emmanuelbaa committed:
Fix(courses): correct checkbox display on different screen sizes

This commit fixes the display of checkboxes in the courses component. It ensures that only one checkbox is displayed per course on larger screens, and that the checkbox is displayed on smaller screens.

emmanuelbaa committed:
Merge branch 'master' into 9162-resources-toggleBoxInconsistency

emmanuelbaa committed:
Reformat .gitignore for consistency

Mutugiii commented:
#9848
